### PR TITLE
Reviews api setup

### PR DESCRIPTION
**Pull request by tchitrakorn**
- This PR contains basic API calls setup (in both client- and server-side) to retrieve reviews
- A helper method for Ratings and Reviews widget (on the server-side) lives in helpers/ratingsReviews.js. This file-structure is subject to change as more methods are added.
- A dummy file containing reviews was added to Ratings and Reviews folder (on the client-side). The need for this file and its location are also subject to change.

### CODE REVIEW

**Comment by tchitrakorn** [comment]
Hi team, here's my PR for the API setup for Ratings and Reviews widget. Could someone take a look at this? Thank you! @matthewwrobel @jabrake

File: client/App.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      'product_id': '22124'
```
Comment (matthewwrobel) [inline client/App.jsx:12]: Did you mean to change the product_id at the app level? Doesn't really matter I guess...

File: client/ratings_reviews/RatingsReviews.jsx [hunk 2]
```diff
@@ -1,9 +1,60 @@
...
+    super(props);
+    this.state = {
+      reviews: reviewsDummy
+    };
+    this.fetchReviews = this.fetchReviews.bind(this);
+  }
+
+  fetchReviews() {
+    $.ajax({
+      url: '/reviews',
```
Comment (matthewwrobel) [inline client/ratings_reviews/RatingsReviews.jsx:21]: Is this a temporary placeholder? I might be missing something :)

**Review by matthewwrobel — COMMENTED** [review]
Lookin' good, Tatt! I think hold off on merging the server stuff until we can talk about it tonight.

File: client/ratings_reviews/RatingsReviews.jsx (hunk 2 above)
Comment (matthewwrobel) [reply client/ratings_reviews/RatingsReviews.jsx:21]: the url property, that is.

**Comment by jabrake** [comment]
Not sure if we got to talk about this PR earlier - but based on our discussion about the new server file, do we still need to follow-up on this? Feel free to close and open a new PR later if you want @tchitrakorn! Also looks like there is a conflict, not sure what it is though?

**Pull request closed without merging**